### DISCUSSION TITLE
Working on the missing Spanish translaton strings

### DISCUSSION THREAD

**Discussion by flrnd** [vendurehq/vendure#1075] [Q&A] [answered]
I'm adding the missing Spanish tranlation strings, if anyone is working on this ping me so we can coordinate, If not, I'll just proceed.

And now, the silly question (Never worked with conventional commits before, sorry :), which type would be this? A bug fix maybe?

Anyway, Great project!

**Comment by michaelbromley** [comment, marked as answer]
Hi @flrnd - just go ahead and do it - I don't know of anyone else wanting to do this right now that needs coordinating.

The silly question is a good one actually and I think I would agree that "fix" is the best label for this.

**Reply by flrnd** [reply]
Perfect, I'll go ahead, Fix it is then!
